intro-HPC/ch_checkpointing.tex

```diff
@@ -1,4 +1,171 @@
 \chapter{Checkpointing}
 \label{ch:checkpointing}
+\section{Why checkpointing?}
+If you want to run jobs that require wall time than the maximum wall time per job
+and/or want to avoid you lose work because of power outages or system crashes,
+you need to resort to checkpointing.
 
-\textit{This chapter is still under construction.}
+\ifgent
+It is highly unlikely that we will offer a queue for jobs that require more
+than 3 days to run, as this makes maintaining the infrastructure more painful.
```

Comment:
@redfast00 Just drop this block, this is not the right place to motivate this...